src/styles/components/replaceserverdetails.less

```diff
@@ -25,10 +25,11 @@
   .table>tbody>tr>th {
     border-top: 0px;
   }
+
   .replace-options {
     margin: 0.5em 0.5em 1.5em 1em;
+    &.more-bottom-margin {
```

Comment:
this smells like a hack, but I'm not good enough with CSS to know how to improve it. Could this instead be handled by using an appropriate CSS selector like `:last-child` or `:last-of-type` or something?

Reply:
will look into it

Reply:
tried it with :last-child which is too generic...if use last-child...it applies to all that kind of last child in the dialog which I don't want that..I am not pro CSS like Jeremy, I will ask Jeremy tomorrow.

Reply:
If those dont work I dont think that the current implementation is all that bad, not worth spending more than 20-30 minutes on

Reply:
Tried it ..since I have various sections contain server-details-container...adding last-child and last-of-type will apply multiple of last-child or last-of-type. I only need add a little bit spaces before the buttons when all the sections are expanded. So will keep the hackish way for now.